Primo Normalized XML.js

```diff
@@ -342,9 +342,11 @@ function stripAuthor(str) {
 	// e.g. Wheaton, Barbara Ketcham [former owner]$$QWheaton, Barbara Ketcham
 	str = str.replace(/^(.*)\$\$Q(.*)$/, "$2");
 	return str
-		// Remove year
-		.replace(/\s*,?\s*\(?\d{4}-?(\d{4})?\)?/g, '')
-		// Remove things like (illustrator). TODO: use this to assign creator type?
+	// Remove year
+		.replace(/\s*,?\s*\(?\d{4}-?(\d{4}|.{3})?\)?/g, '')
+		// Remove creator type like (illustrator)
+		.replace(/\s*,?\s*[[(][^()]*[\])]$/, '')
+		// Sometimes creator type is displayed twice
```

Comment:
Couldn't you just do `/(\s*,?\s*[[(][^()]*[\])])+$/` instead of duplicating the same line?

Reply:
Yes, you right.